### PR TITLE
Make changes to ``gtf-to-bed`` process

### PR DESCRIPTION
**Pull request by marcellevstek**
* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [x] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [x] All inputs are used in process.
* [x] All output fields have a value assigned to them.

### CODE REVIEW

File: resolwe_bio/processes/support_processors/gtf_to_bed.py [hunk 1]
```diff
@@ -23,9 +23,9 @@ class GTFtoBED(Process):
     slug = "gtf-to-bed"
     name = "GTF to BED"
     process_type = "data:bed"
-    version = "1.1.0"
+    version = "1.2.0"
     category = "Other"
-    data_name = "Converted GTF to BED file"
+    data_name = "{{ geneset|name|default('?') }}"
```
Comment (jkokosar) [inline resolwe_bio/processes/support_processors/gtf_to_bed.py:28]: `geneset` is not a required input, thus the name variable will not always be constructed as intended. It is best to find some other informative way to set the Data object name.

Comment (marcellevstek) [reply resolwe_bio/processes/support_processors/gtf_to_bed.py:28]: Thanks, looks like I missed it. We should consider making geneset a required input. I don't see a use case when you want to extract the whole exome or transcriptome, as you can obtain those files from other sources.

Comment (jkokosar) [reply resolwe_bio/processes/support_processors/gtf_to_bed.py:28]: If this is the intended use case, then please make it so. Make sure to highlight this requirement in the process description as well.

**Review by jkokosar — APPROVED** [review]
LGTM. Please squash and rebase when the previous PR is merged.

**Pull request merged**